plugins/doc_fragments/eda_controller.py

```diff
@@ -10,37 +10,43 @@ class ModuleDocFragment:
     controller_host:
         description:
           - The URL of the EDA controller.
-          - If not set, the value of the C(CONTROLLER_HOST) environment variable will be used.
+          - If not set, the value of C(CONTROLLER_HOST), or E(AAP_HOSTNAME) environment variables will be used.
```

Comment:
Does it make sense to you to add a note like "Support for E(AAP_HOSTNAME) has been added in version 2.x?"

Reply:
Done! I added another line to address this in each auth option